typing_extensions/setup.py

```diff
@@ -53,14 +53,14 @@
 
 install_requires = []
 if sys.version_info < (3, 5):
```

Comment:
I wonder if this could be done the way we do it for mypy now?
```
      extras_require = {
          ':python_version < "3.5"': 'typing >= 3.5.3',
      },
```
(Except using 3.6.2 for the typing version.)

IIUC that way you can build universal wheels that do the right thing.